### PR TITLE
Dpys

### PR DESCRIPTION
**Pull request by dPys**
*Simplify DSN

See most recent discussion:
https://github.com/nipy/dipy/pull/1900

1 additional tweak may still ultimately be needed if this doesn't raise discriminability, so let's keep an eye...

@dPys

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
         "pybids==0.6.4",
         "setuptools>=40.0",
         "scikit-image",
-        "networkx==1.9",
+        "networkx",
```
Comment (loftusa) [inline setup.py:59]: Only one comment -- have we checked to make sure the 1.9 networkx code in `gen_graph.py` is compatible with this?

**Review by loftusa — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (dPys) [reply setup.py:59]: No, will check that now. Good catch @loftusa

Comment (dPys) [reply setup.py:59]: yep, works fine with networkx==2.2

**Comment by loftusa** [comment]
you merged when travis didn't pass my dude, it shows native_dsn broken in this branch

**Pull request merged**